exercises/levenshtein-distance/canonical-data.json

```
@@ -0,0 +1,65 @@
+{
+  "exercise": "levenshtein-distance",
+  "version": "1.0.0",
+  "cases": [
+    {
+      "description": "Calculate Levenshtein distance",
+      "cases": [
+        {
+          "description": "mine_to_times",
+          "property": "distance",
+          "input": {
+            "phrase": "mine",
+			"phrase": "times"
```

```suggestion
            "phrase": "times"
```
Replaces tabs with spaces.